code/modules/mob/living/silicon/pai/pai.dm

```diff
@@ -34,7 +34,8 @@
 		"Parrot" = "parrot",
 		"Rabbit" = "rabbit",
 		"Bear" = "bear",  //VOREStation Edit
-		"Raccoon" = "raccoon" //Chompstation Edit
+		"Raccoon" = "raccoon", //Chompstation Edit

```

Comment:
Still might want to add a comma at the end of 'Edit' in the comments here, since it's still well within a list.